WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailToolbar.swift

```diff
@@ -183,7 +183,8 @@ class ReaderDetailToolbar: UIView, NibLoadable {
             return
         }
 
-        likeButton.isEnabled = (ReaderHelpers.isLoggedIn() || post.likeCount.intValue > 0) && !post.isExternal
+        let postLikeCount = post.likeCount?.intValue ?? 0
```

Comment:
Is this the same value as line `189`?

`let likeCount = post.likeCount()?.intValue ?? 0`

Reply:
It is.

Changed in 63403d140120d5f21c39890a0caa12b8209efd93